@xen-orchestra/web-core/lib/components/state-hero/VtsStateHero.vue

```diff
@@ -1,8 +1,11 @@
 <template>
   <div :class="type" class="vts-state-hero">
     <UiLoader v-if="busy" class="loader" />
-    <img v-else-if="imageSrc" :src="imageSrc" alt="" class="image" />
-    <p v-if="slots.default" :class="typoClass" class="text">
+    <p v-if="slots.default && type === 'panel' && !busy" :class="typoClass" class="text">
+      <slot />
+    </p>
+    <img v-if="imageSrc && !busy" :src="imageSrc" alt="" class="image" />
+    <p v-if="slots.default && type !== 'panel' && !busy" :class="typoClass" class="text">
       <slot />
     </p>
```

Comment:
To keep readability and avoid code duplication, I think we can leverage `flex` instead:
```suggestion
    <UiLoader v-if="busy" class="loader" />
    <img v-else-if="imageSrc" :src="imageSrc" alt="" class="image" />
    <p v-if="slots.default" :class="typoClass" class="text">
      <slot />
    </p>
```
Then, in the CSS:
```css
.vts-state-hero {
  /* Other styles */

  &.page {
    /* Other styles */

    .image {
      order: 2;
      /* Other styles */
    }

    .loader {
      order: 1;
      /* Other styles */
    }

    .text {
      order: 3;
    }
  }

  &.card {
    /* Other styles */

    .image {
      order: 2;
      /* Other styles */
    }

    .loader {
      order: 1;
      /* Other styles */
    }

    .text {
      order: 3;
    }
  }

  &.panel {
    /* Other styles */

    .text {
      order: 2;
    }

    .image {
      order: 3;
      /* Other styles */
    }

    .loader {
      order: 3;
      /* Other styles */
    }
  }
}

```